mucommander-core/src/main/java/com/mucommander/ui/viewer/FileEditorPresenter.java

```diff
@@ -46,6 +46,8 @@
 import com.mucommander.viewer.FileEditor;
 import com.mucommander.viewer.FileEditorService;
 
+import org.slf4j.LoggerFactory;
```

Comment:
yeah, I've also noticed that we're inconsistent with the logger we use - but that's unused now, right?